application/models/UserGroup.php

```diff
@@ -590,6 +590,7 @@ public function sendUserEmails($ugid, $subject, $body, $copy)
             if ($emaiLResult['success']) {
                 $msgToUser .= gT('Sending successful') . "<br>";
             } else {
+                throw new Exception("Failed to send mail");
                 $msgToUser .= gT('Error: ') . $emaiLResult['msg'] . "<br>";
```

Comment:
This line is dead code, can be removed.

Reply:
without this line, error is not catched

https://github.com/LimeSurvey/LimeSurvey/blob/91db5ff516f75b4f947c929aebbbf5220620d0a7/application/controllers/UserGroupController.php#L475